File: src/SOP8/Base/Base.Model/Common/Team/TemporaryMember.cs
using System;
using dnsDapperDBUtil.DataAccessLayer.IDAL;

namespace Base.Model.Common.Team
{
	public class TemporaryMember : Table
	{
		public enum Fields { tmpr_memb_sn, disp_name, tmpr_sn, rgl_sn, rgl_memb_sn, role_optn_no, role_no, memo };
		public enum WriteFields { disp_name, tmpr_sn, rgl_sn, rgl_memb_sn, role_optn_no, role_no, memo };

		public int tmpr_memb_sn { get; set; }
		public string/* nullable */ disp_name { get; set; }
		public int tmpr_sn { get; set; }
		public int? rgl_sn { get; set; }
		public int? rgl_memb_sn { get; set; }
		public int? role_optn_no { get; set; }
		public int? role_no { get; set; }
		public string/* nullable */ memo { get; set; }

		public static string TableName { get { return "co_team_tmpr_memb"; } }

		public override string GetTableName()
		{
			return TableName;
		}

		public override string GetPrimaryCondition()
		{
			return string.Format("{0} = {1}", Fields.tmpr_memb_sn, tmpr_memb_sn);
		}

		public override Type GetFieldType()
		{
			return typeof(Fields);
		}

		public override Type GetWriteFieldType()
		{
			return typeof(WriteFields);
		}

		public void FromCopy(TemporaryMember obj)
		{
			this.tmpr_memb_sn = obj.tmpr_memb_sn;
			this.disp_name = obj.disp_name;
			this.tmpr_sn = obj.tmpr_sn;
			this.rgl_sn = obj.rgl_sn;
			this.rgl_memb_sn = obj.rgl_memb_sn;
			this.role_optn_no = obj.role_optn_no;
			this.role_no = obj.role_no;
			this.memo = obj.memo;
		}
	}
}

File: src/SOP8/광양/EgovLogin/base-model/src/main/java/egovframework/base/model/common/team/TemporaryMember.java
package egovframework.base.model.common.team;

import lombok.*;
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class TemporaryMember {
	public enum Fields { tmpr_memb_sn, disp_name, tmpr_sn, rgl_sn, rgl_memb_sn, role_optn_no, role_no, memo }

	private int tmpr_memb_sn;

	private String disp_name;

	private int tmpr_sn;

	private Integer rgl_sn;

	private Integer rgl_memb_sn;

	private Integer role_optn_no;

	private Integer role_no;

	private String memo;

	public static final String autoIncreaseField = "tmpr_memb_sn";
	public static final String tableName = "co_team_tmpr_memb";
	public static final String[] primaryKeys = { "tmpr_memb_sn" };

}
